SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -45,6 +49,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (ConfigRestartRequired = false, DisplayName = "Actor Replication Rate Limit"))
 	uint32 ActorReplicationRateLimit;
 
+	/** QBI is required for level streaming to be supported when using spatial networking, however comes at a performance cost for larger-scale projects.*/
+	UPROPERTY(EditAnywhere, config, Category = "QBI", meta = (ConfigRestartRequired = false, DisplayName = "QBI Enabled"))
+	bool bUsingQBI;
+
 	virtual FString ToString();
```

Comment:
Not a new addition in this PR, but we should avoid interleaving functions and fields.